ena-submission/pyproject.toml

```diff
@@ -21,3 +21,25 @@ build-backend = "hatchling.build"
 
 [tool.hatch.build.targets.wheel]
 packages = ["src/ena_deposition"]
+
+[tool.pytest.ini_options]
+# Live logging - show INFO+ during execution
+log_cli = true
+log_cli_level = "INFO"  # Show INFO logs during execution
+log_cli_format = "%(asctime)s [%(levelname)8s] %(name)s: %(message)s (%(filename)s:%(lineno)d)"
+log_cli_date_format = "%Y-%m-%d %H:%M:%S"
+
+# File logging - captures everything for detailed analysis
+log_file = "tests.log"
```

Comment:
do we have to worry about this getting too large on production?

Reply:
oh sorry this is just for tests!

Reply:
Yeah this is the full log stream for reviewing what happened during tests - it's also in gitignore